<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_form</class>
 <widget class="QWidget" name="main_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart Refrigerator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_current_temp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Current Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fridge_temp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fridge Temperature:      -7째C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_freezer_temp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Freezer Temperature:   -15째C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp_plan_high">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--째C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp_plan_down">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--째C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_temp_plan_plus">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_temp_plan_minus">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fact:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Standart:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
